extensions/ql-vscode/src/databases/db-config.ts

```diff
@@ -1,10 +1,20 @@
 // Contains models for the data we want to store in the database config
 
 export interface DbConfig {
+  databases: DbConfigDatabases;
+  selected?: SelectedDb;
+}
+
+export interface DbConfigDatabases {
   remote: RemoteDbConfig;
   local: LocalDbConfig;
 }
 
+export interface SelectedDb {
+  kind: string;
+  path: string;
```

Comment:
Since this can either be a path or a value I think we decided on the term `value`. Let's also add more specific types for this (happy to pair if it helps).